<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayWidget</class>
 <widget class="QWidget" name="PlayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>670</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>670</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>670</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(240, 251, 255)</string>
  </property>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>600</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border:2px solid #124d92;
	border-radius:5px;
	background-color:rgb(167, 255, 197);
}
QPushButton:pressed {
	background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="restartButton">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>540</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border:2px solid #124d92;
	border-radius:5px;
	background-color:rgb(167, 255, 197);
}
QPushButton:pressed {
	background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
   </property>
   <property name="text">
    <string>重新开始</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>410</y>
     <width>411</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="firstStackedWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page11">
       <widget class="QPushButton" name="callButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>110</width>
          <height>50</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border:2px solid #124d92;
	border-radius:5px;
	background-color:rgb(167, 255, 197);
}
QPushButton:pressed {
	background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
        </property>
        <property name="text">
         <string>叫地主</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page12">
       <widget class="QPushButton" name="playButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>111</width>
          <height>50</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border:2px solid #124d92;
	border-radius:5px;
	background-color:rgb(167, 255, 197);
}
QPushButton:pressed {
	background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
        </property>
        <property name="text">
         <string>出牌</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="myStatus">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 0, 127)</string>
      </property>
      <property name="text">
       <string>我在这</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="secondStackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page21">
       <widget class="QPushButton" name="notCallButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>20</y>
          <width>111</width>
          <height>50</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border:2px solid #124d92;
	border-radius:5px;
	background-color:rgb(167, 255, 197);
}
QPushButton:pressed {
	background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
        </property>
        <property name="text">
         <string>不叫</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page22">
       <widget class="QPushButton" name="notPlayButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>20</y>
          <width>111</width>
          <height>50</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border:2px solid #124d92;
	border-radius:5px;
	background-color:rgb(167, 255, 197);
}
QPushButton:pressed {
	background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
        </property>
        <property name="text">
         <string>不要</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="myMessage">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>360</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>我在这</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="secondMessage">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>40</width>
     <height>120</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>我在这</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="firstMessage">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>250</y>
     <width>40</width>
     <height>120</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>我在这</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="showPic1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0,0,0,0)</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="showPic2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0,0,0,0)</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="showPic3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0,0,0,0)</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="secondStatus">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>40</width>
     <height>120</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 127)</string>
   </property>
   <property name="text">
    <string>我在这</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="firstStatus">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>250</y>
     <width>40</width>
     <height>120</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 127)</string>
   </property>
   <property name="text">
    <string>我在这</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="secondCardsNum">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>61</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正舒体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(255, 0, 127);
background-color: rgb(209, 205, 255);</string>
   </property>
   <property name="text">
    <string>17</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="firstCardsNum">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>20</y>
     <width>61</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正舒体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(255, 0, 127);
background-color: rgb(209, 205, 255);</string>
   </property>
   <property name="text">
    <string>17</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
